python/cm/job.py

```diff
@@ -187,12 +187,11 @@ def check_action_config(
         return {}, {}
     if not conf:
         err("TASK_ERROR", "action config is required")
-    obj_conf, current_attr = None, None
+    obj_conf = None
     if obj.config:
-        cl = ConfigLog.objects.get(obj_ref=obj.config, id=obj.config.current)
-        obj_conf = cl.config
-        current_attr = cl.attr
```

Comment:
It is necessary to completely remove this block, and pass the configuration of the action to the function `variant.process_variant(obj, spec, conf)`

Reply:
done